src/pages/education.tsx

```diff
@@ -0,0 +1,218 @@
+                            />
+                            <path className="st0" d="M758.72,531.24l152.68,95.87l-35.04,55.8c-1.98,7.37,1.45,15.65,8.89,20.08c14.3,8.48,26.38-9.46,51.5,6.31	c20.83,13.08,27.12,40.57,14.03,61.4c-13.08,20.83-40.57,27.12-61.4,14.03c-22.91-14.38-14.67-32.62-29.46-41.41	c-5.43-3.23-15.13-3.04-20.89-0.97l-34.69,55.25l-59.56-37.39c-0.34-4.84,0.32-10.09,2.4-13.59c8.78-14.79,27.02-6.55,41.4-29.46	c13.08-20.83,6.8-48.32-14.03-61.4c-20.84-13.08-48.32-6.8-61.4,14.03c-15.77,25.12,2.17,37.19-6.31,51.49	c-2.52,4.24-6.28,7.17-10.42,8.55l-62.57-39.29l95.87-152.68C735.87,528.07,748.92,525.09,758.72,531.24L758.72,531.24z"
+                            />
+                            <path className="st0" d="M656.52,1177.51l35.26-56.18c2.14-7.32-1.1-15.68-8.44-20.27c-14.11-8.8-26.58,8.87-51.34-7.46	c-20.54-13.54-26.2-41.16-12.66-61.7c13.54-20.54,41.16-26.2,61.7-12.66c22.58,14.89,13.94,32.94,28.53,42.05	c5.36,3.35,15.06,3.37,20.86,1.44l35.57-54.16"
+                            />
+                            </svg>
+                        </div>
+                    </Hero>
+                </Container>
+                <Container>
+                    <SectionFeatures>
+                        <h3>How to enhance the learning process with Gitpod</h3>
+                            <div className="features">
+                                <div className="feature">
+                                    <h4>No setup</h4>
+                                    <p>Don't waste time on tedious setup. With Gitpod developer enviroments are automated, reproducible and versioned.</p>
+                                </div>
+                                <div className="feature">
+                                    <h4>Works on every device</h4>
+                                    <p>With Gitpod coding gets accessible for  everyone your students can contribute with a Chromebook or a tablet.</p>
+                                </div>
+                                <div className="feature">
+                                    <h4>Enhances the workflow</h4>
+                                    <p>Create customized lessons for your students. Everyone can start working on new asignments within minutes.</p>
+                                </div>
+                            </div>
+                            <div className="features">
+                                <div className="feature">
+                                    <h4>Collaborative</h4>
+                                    <p>Gitpod strengthens collaboration between your students. Code reviews has never been this easy and sharable snapshots foster the team workflow.</p>
+                                </div>
+                                <div className="feature">
+                                    <h4>Continous Support</h4>
+                                    <p>Easy maintenance, flexible subscriptions and continous support if needed.</p>
```

Comment:
Suggestion (support if needed == support):

```suggestion
                                    <p>Easy maintenance, flexible subscriptions and continuous support.</p>
```